test/test_cli.rb

```diff
@@ -93,7 +93,7 @@ def test_control_for_ssl
 
     expected_stats = /{"started_at":"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z","backlog":0,"running":0,"pool_capacity":16,"max_threads":16,"requests_count":0}/
     assert_match(expected_stats, body.split(/\r?\n/).last)
-    assert_equal([:started_at, :backlog, :running, :pool_capacity, :max_threads, :requests_count], Puma.stats.keys)
+    assert_equal(JSON.parse(body.split(/\r?\n/).last, symbolize_names: true), Puma.stats)
```

Comment:
We already checked the entire response on the previous line. IMO, this test is redundant.